cosmwasm/ibc-union/core/Cargo.toml

```diff
@@ -19,10 +19,12 @@ library = []
 
 [dependencies]
 alloy                         = { workspace = true, features = ["sol-types"] }
+bincode                       = { workspace = true, features = ["alloc", "derive"], optional = true }
 cosmwasm-schema               = { workspace = true }
 cosmwasm-std                  = { workspace = true, features = ["abort"] }
 cw-storage-plus               = { workspace = true }
 ethabi                        = { workspace = true }
+ethereum-light-client-types   = { workspace = true, features = ["serde", "bincode"] }
```

Comment:
this may be resolved as i review the rest of the pr: why is this here?

Reply:
yeah why is this here?

Reply:
this shouldnt be here, removed